volume/mounts/mounts.go

```diff
@@ -188,24 +184,6 @@ func (m *MountPoint) Setup(mountLabel string, rootIDs idtools.Identity, checkFun
 
 			clean = safePath.Close
 			volumePath = safePath.Path()
-
-			// TODO: Find a way to have it only for integration tests
-			if strings.HasSuffix(subpath, "iwanttobehackedwithtoctou") {
```

Comment:
It would be nice to keep the TOCTOU test – I wonder if we can find a way to keep it without touching the implementation.

Reply:
Yeah, I have no idea how to do it nicely with the current way integration tests are run.
We still have the `TestJoinWithSymlinkReplace` unit test that checks it though.